R/package_activity_list.R

```diff
@@ -9,10 +9,12 @@
 #' package_activity_list("34d60b13-1fd5-430e-b0ec-c8bc7f4841cf", as="table")
 #' package_activity_list("34d60b13-1fd5-430e-b0ec-c8bc7f4841cf", as="json")
 #' }
-package_activity_list <- function(id, offset = 0, limit = 31, url = get_ckanr_url(),
+package_activity_list <- function(id,
+                                  offset = 0,
+                                  limit = 31,
+                                  url=get_default_url(),
                                   as="list", ...) {
```

Comment:
OK, will undo. I did this to parameter lists longer than one line. I'll reflow them but still line break them at 80 chars if that's OK.